bugbot/rules/crash_small_volume.py

```diff
@@ -124,6 +124,10 @@ def handle_bug(self, bug, data):
             "signatures": signatures,
         }
 
+        # Add needinfo IDs only if the keyword to remove is "topcrash"
+        if "topcrash" in keywords_to_remove:
+            data[bugid]["needinfo_ids"] = self.get_needinfo_topcrash_ids(bug)
+
```

Comment:
It would be neater to always pass a list; if the `topcrash` were not removed, you could pass an empty list.

Reply:
Done in feef346863acd08cad0b02f7f196404ec4849838

Reply:
I mean something like:
```py
"needinfo_ids": self.get_needinfo_topcrash_ids(bug) if "topcrash" in keywords_to_remove else []
```
With that, you do not need to use `.git()` or check for `topcrash` when clearing the flags.

Reply:
Fixed in e5b32720318455199ae6ac3ae29fc39c57eca244.